### PR TITLE
Add an activity event to track when when the Dashboard page loads

### PR DESCRIPTION
**Pull request by johnclary**
## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/15392

## Testing
**URL to test:** Local
You will use your SQL client to inspect activity events as they are logged.

1. Start Moped locally. Log out, so that you're looking at the login screen.
2. Open your SQL client, and delete any records in the `moped_user_events` table:

```sql
DELETE FROM moped_user_events WHERE TRUE;
``` 
3. Login to Moped. There should now be one event in the `moped_user_events` table. The event name should be `app_load`.
4. Navigate to the **Dashboard** page. There should now be a second event in the `moped_user_events` table with the name `dashboard_load`.
5. While still on the **Dashboard** page, refresh the page in your browser. This will create two new events: `app_load` and `dashboard_load`.
6. Now navigate to the **Staff** page and verify that no new activity events have been created.
 

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

File: moped-database/migrations/1706117943942_add_dashboard_load_event/up.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+ALTER TABLE moped_user_events DROP CONSTRAINT moped_user_events_event_name_check;
+
+ALTER TABLE moped_user_events
+    ADD CONSTRAINT moped_user_events_event_name_check CHECK (event_name in('app_load', 'dashboard_load'));
```
Comment (mddilley) [inline moped-database/migrations/1706117943942_add_dashboard_load_event/up.sql:4]: ✨

**Review by mddilley — CHANGES_REQUESTED** [review]
Love seeing the tracker grow so easily to handle new events!! 🚀 🙌 The down migration doesn't work as is, but, imo, we move forward and never go back! 🏎️💨💨 Probably need a comment that there is no going back? Not sure how we want to handle that.

**Review by chiaberry — APPROVED** [review]
I'm looking forward to finding out who uses the dashboard page and how often

File: moped-database/migrations/1706117943942_add_dashboard_load_event/down.sql [hunk 2]
```diff
@@ -0,0 +1,6 @@
+DELETE FROM moped_user_events where event_name = 'dashboard_load';
```
Comment (mddilley) [inline moped-database/migrations/1706117943942_add_dashboard_load_event/down.sql:1]: Cool, that makes sense to me! 🙏

**Review by mddilley — APPROVED** [review]
Ready to go and to start collecting new events. 👀 🙌

**Comment by johnclary** [comment]
🍿

**Pull request merged**